+ source /root/mugen/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Sat Apr 29 07:08:49 2023 - INFO  - Start to prepare the test environment.
+ DNF_INSTALL aide
+ pkgs=aide
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen/libs/locallibs/rpm_manage.py install --pkgs aide --node 1 --tempfile ''
+ tmpfile2=/tmp/tmpnsih8oq6
+ '[' -z '' ']'
+ tmpfile=/tmp/tmpnsih8oq6
+ sed -i '$a/home DATAONLY' /etc/aide.conf
+ aide --init
Start timestamp: 2023-04-29 07:09:20 +0800 (AIDE 0.17.4)
AIDE initialized database at /var/lib/aide/aide.db.new.gz

Number of entries:	75443

---------------------------------------------------
The attributes of the (uncompressed) database(s):
---------------------------------------------------

/var/lib/aide/aide.db.new.gz
 MD5       : Gvgl6mXKSaTvdcphnlyWgg==
 SHA1      : DaW77EGAd2rwYaVG0kMDWhqn8ds=
 SHA256    : OtGcwanddmg1QvMFUJRkBKSjka928Ume
             bdLUrqU3uMg=
 SHA512    : rqExJ66A+XQzTVozTGQtf7aiJk2fAFsv
             RSOvHPghnZpMr23cv5IJWusujwWyCYZ4
             pYIKLjs/jrn/xFly+7A/7w==
 RMD160    : Kvcu2bWdKooeWtYTZeNhewCb3qg=
 TIGER     : YGSv7MY24nQv4GNtbSdNeaDY3gyqdyir
 CRC32     : WR0Klg==
 WHIRLPOOL : gZ7Yf5eCMO1iyVsqcj8BGUaFj+KfJw9N
             j11oPPCDv6HGxtw4SCwcOquU0tgMFLIe
             ukWs5xVtw65cARrpoyexiw==
 GOST      : 48j38ft2fgIM1gEzX62gPRIhL1806orx
             sX83rguguBw=
 STRIBOG256: Q031xvIQHLXuS+kzXrzVT9tCVONdODi2
             kOEXV6wrenQ=
 STRIBOG512: wrOGnczOQt6T+q2AslR0hL0EL7M3Q9z+
             2NIfniSWlL4GY/YVfwFFcVve+9uoMenG
             MpZppNkyw7X8l+yuyABVRA==
 SM3       : tJWcp/hCs+Y3m9O51qU7JfZ+hnhRRzSB
             xQJui4VmLAw=


End timestamp: 2023-04-29 07:15:38 +0800 (run time: 6m 18s)
+ LOG_INFO 'End to prepare the test environment.'
+ message='End to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to prepare the test environment.'
Sat Apr 29 07:15:39 2023 - INFO  - End to prepare the test environment.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Sat Apr 29 07:15:40 2023 - INFO  - Start to run test.
+ ls -l /etc/aide.conf
+ awk '{print $3}'
+ grep root
root
+ CHECK_RESULT 0 0 0 'Check aide.conf file owner failed!'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check aide.conf file owner failed!'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ ls -l /etc/aide.conf
+ awk '{print $4}'
+ grep root
root
+ CHECK_RESULT 0 0 0 'Check aide.conf file group failed!'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check aide.conf file group failed!'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ ls -l /etc/aide.conf
+ awk '{print $1}'
+ cut -b 2-10
+ grep rw-------
rw-------
+ CHECK_RESULT 0 0 0 'Check aide.conf file permission failed!'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check aide.conf file permission failed!'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ ls -l /var/log/aide/aide.log
+ awk '{print $3}'
+ grep root
root
+ CHECK_RESULT 0 0 0 'Check aide.log file owner failed!'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check aide.log file owner failed!'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ ls -l /var/log/aide/aide.log
+ awk '{print $4}'
+ grep root
root
+ CHECK_RESULT 0 0 0 'Check aide.log file group failed!'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check aide.log file group failed!'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ ls -l /var/log/aide/aide.log
+ cut -b 2-10
+ awk '{print $1}'
+ grep rw-------
rw-------
+ CHECK_RESULT 0 0 0 'Check aide.log file permission failed!'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check aide.log file permission failed!'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Sat Apr 29 07:15:41 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z ''
+ test 0 -eq 0
+ LOG_INFO 'succeed to execute the case.'
+ message='succeed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'succeed to execute the case.'
Sat Apr 29 07:15:42 2023 - INFO  - succeed to execute the case.
+ exec_result=
+ exit 0
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Sat Apr 29 07:15:42 2023 - INFO  - Start to restore the test environment.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmpnsih8oq6 ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmpnsih8oq6
+ '[' 0 -ne 0 ']'
+ rm -rf /var/log/aide/
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Sat Apr 29 07:15:56 2023 - INFO  - End to restore the test environment.
